docs/pull_request_template.md

```diff
@@ -55,13 +55,13 @@ all boxes that apply at the bottom.
    provided with this PR, within the PR or in future PRs.
    
 
-**Please review and check the following**:
-<!-- replace the empty checkboxes [ ] below with checked ones [x] accordingly -->
+Please review and check the following:
+(Replace the empty checkboxes [ ] below with checked ones [x] accordingly) -->
 
   - [ ] I have read the [guidelines for contributions](https://github.com/poldracklab/fmriprep/blob/master/CONTRIBUTING.md).
   - [ ] I understand that my contributions will not be merged unless the work is
         finished (i.e. no ``[WIP]`` tag remains in the title of my PR) and tests pass.
   - [ ] The proposed code follows the
```

Comment:
Can we remove these two line breaks (64 and 65). Apparently, markdown does not like them (and fixing indentations won't help)